src/adagrams.js

```diff
@@ -1,15 +1,95 @@
+import { letterPool, scoreChart } from 'constants';
+
+// helper function to create letter pool array
+export const createLetterPool = () => {
```

Comment:
Great use of a helper function! 

If our function will only be used inside the file it's defined in, we can remove the `export`, since that is needed for other files to know about the function.